src/Microsoft.VisualBasic/tests/VB/Dummy.vb

```diff
@@ -6,16 +6,19 @@ Imports Microsoft.VisualBasic
 Imports System
 Imports Xunit
 
-Namespace Microsoft.VisualBasic.Tests
+Namespace Microsoft.VisualBasic.Tests.VB
 
-    Public Class Dummy
+    Public NotInheritable Class DummyTest
 
         <Fact>
-        Public Sub Dummy()
-
-            Dim dateTimeNowBefore As DateTime = DateTime.Now()
-            Dim now As DateTime = DateAndTime.Now()
-            Dim dateTimeNowAfter As DateTime = DateTime.Now()
+        Public Shared Sub Dummy()
+            'While (Not System.Diagnostics.Debugger.IsAttached)
+            '    System.Threading.Thread.Sleep(1000)
+            'End While
+
+            Dim dateTimeNowBefore As Date = Date.Now()
+            Dim now As Date = DateAndTime.Now()
+            Dim dateTimeNowAfter As Date = Date.Now()
 
             Assert.InRange(now, dateTimeNowBefore, dateTimeNowAfter)
```

Comment:
Are the changes to this file needed? If not, please revert.

Reply:
They were in the original code I started with from Dan M. I needed to comment out to get tests to run. If I delete them people looking to debug Vb code in VS will not have an example.

Reply:
See https://github.com/dotnet/corefx/issues/31181#issuecomment-414104137 for explaination